src/main/java/core/db/DBUpdater.java

```diff
@@ -76,6 +77,46 @@ void updateDB(int DBVersion) {
 		}
 	}
 
+	private final HashMap<String, PreparedStatement> migrateStatements = new HashMap<>();
+	private PreparedStatement getMigrateEscapesStatement(String table, String column){
+		var sql = "Update "+ table + " SET " + column + "=REPLACE(" + column + ", ?, ?)";
+		var ret = migrateStatements.get(sql);
+		if ( ret == null ){
+			ret = new DBManager.PreparedStatementBuilder(sql).getStatement();
+			migrateStatements.put(sql, ret);
+		}
+		return ret;
+	}
+	private PreparedStatement getMigrateEscapesStatement(String table, String column, String where){
+		var sql = "Update "+ table + " SET " + column + "=REPLACE("+column+", ?, ?) " + where;
+		var ret = migrateStatements.get(sql);
+		if ( ret == null ){
+			ret = new DBManager.PreparedStatementBuilder(sql).getStatement();
+			migrateStatements.put(sql, ret);
+		}
+		return ret;
+	}
+	private void migrateEscapes(String table, String ... columns){
```

Comment:
There is code duplication here, this method can probably be written using the other one.

So replacements are:
- `§` -> `\`
- `#` -> `'`

Correct?

Reply:
you're right - changed this.